boundary/analyze.py

```diff
@@ -66,6 +61,11 @@ def get_in_any(fn, files):
     return None
 
 def find_in_vmlinux(vmlinux_elf):
+    # store sympos for local functions in module files
+    local_sympos = {}
+    # store exported function symbol (EXPORT_SYMBOL, EXPORT_SYMBOL_GPL)
+    export_func = set()
+    multvers = set()
```

Comment:
It's not multi-version, but name mangling. Please use name mangling in commit log.